.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
-_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._
+_If any of the below don't apply to this Pull Request, mark the checkbox as done._
 
 - [ ] I have tested my code.
 - [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
 - [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
   - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
-  - If changes are requested, I will re-request the review after making them.
\ No newline at end of file
+  - If changes are requested, I will re-request the review after making the changes.
```

Comment:
?

Reply:
What? It sounds like I should re-request the review after making the review. It's a bit ambiguous which them it is referring to.